src/main/java/taks4/BigLetters.java

```diff
@@ -0,0 +1,8 @@
+package taks4;
+
+public class BigLetters implements TextFormatter {
+    @Override
+    public String formatText(String text) {
+        return text.toUpperCase();
```

Comment:
To jest ten moment kiedy jesteś na takim stopniu zaawansowania że powinieneś tutaj brać pod uwagę, że zmienna text będzie nullem. co wtedy? spróbuj napisać sobie taki test gdzie przekażesz do tej metody (lub tej pomniejszającej litery) null zamiast "jakiegoś tekstu" zobacz co się stanie i pomyśl jak to rozwiązać. odsyłam tu: https://www.tutorialspoint.com/check-if-a-string-is-empty-or-null-in-java